frontend/src/advising/Templates/TemplateInfoPage.tsx

```diff
@@ -65,7 +70,16 @@ export const NewTemplatesPage: React.FC = () => {
   ];
   const coopCycles = useSelector((state: AppState) => getPlansFromState(state));
   const buttonSize = 90;
-  const disabled = !(name && major && catalogYear && coopCycle);
+
+  const majorObj = findMajorFromName(major, majors, Number(catalogYear));
+
+  const onClickNext = () => {
```

Comment:
I think you can leave it as disabled instead (since otherwise you'll need error displaying for every incorrect field, not just concentration)